src/Symfony/Bundle/FrameworkBundle/Command/RouterDebugCommand.php

```diff
@@ -125,4 +131,22 @@ private function findRouteNameContaining(string $name, RouteCollection $routes):
 
         return $foundRoutesNames;
     }
+
+    private function sortRoutes(RouteCollection $routes, string $propertyName): RouteCollection
+    {
+        $sortedRoutes = $routes->all();
+        if ('name' === $propertyName) {
```

Comment:
I think a `switch` statement would be a better fit here

Reply:
Ok I'm going to change this